### ISSUE TITLE
Make it reusable

### ISSUE THREAD

**Issue by joshmackwilliams** [Voltron-UTD/voltron_steering_pid#3]
Currently, this code is hardwired to be a PID controller for steering only. It could e a more general PID controller (that could also work for acceleration, for instance), but it would need to accept incoming floats. This would require a separate translator node that would pull out the wheel angle from incoming vehicle commands. Additionally, other issues like the fact that it currently takes a hardcoded "max steering angle" would have to be generalized. Topic remappings would be used to ensure that the controllers run in the right place

TLDR: There's no reason to write another PID for brake and gas. Rewrite this one so it can work for either.
